scout/server/blueprints/variants/views.py

```diff
@@ -229,24 +248,53 @@ def sv_variants(institute_id, case_name):
             'gene_panels': default_panels
              })
 
+    user_obj = store.user(current_user.email)
     if(request.method == "POST"):
         if bool(request.form.get('clinical_filter')):
             form = SvFiltersForm(clinical_filter)
             form.csrf_token = request.args.get('csrf_token')
+        elif bool(request.form.get('save_filter')):
+            # The form should be applied and remain set the page after saving
+            form = SvFiltersForm(request.form)
+            # Stash the filter to db to make available for this institute
+            filter_obj = request.form
+            store.stash_filter(filter_obj, institute_obj, case_obj, user_obj, category='sv')
+        elif bool(request.form.get('load_filter')):
+            filter_display_name = request.form.get('filters')
+            filter_obj = store.retrieve_filter(filter_display_name)
+            form = SvFiltersForm(MultiDict(filter_obj))
         else:
             form = SvFiltersForm(request.form)
     else:
         form = SvFiltersForm(request.args)
 
+    # populate filters dropdown
+    available_filters = store.filters(institute_id, category='sv')
+    form.filters.choices = [(filter.get('_id'), filter.get('display_name'))
+        for filter in available_filters]
+
+    # redundant?
     form.variant_type.data = variant_type
```

Comment:
Dunno..?

Reply:
I have a feeling @moonso is itching to deal with that in a little bit, so I'll leave it as a cliff-hanger.